pkg/trace/config/apply.go

```diff
@@ -88,6 +93,13 @@ type Enablable struct {
 	Enabled bool `mapstructure:"enabled"`
 }
 
+// SQLObfuscationConfig holds the obfuscation configuration for SQL.
+// easyjson:json
+type SQLObfuscationConfig struct {
```

Comment:
This is now part of `datadog.yaml` because of it being added here. You also need to set it as "known" in `pkg/config/apm.go`, add an environment variable, and make sure it is actually picked up (currently only the feature flag is). It also needs to be documented in `config_template.yaml`

Reply:
Updated: https://github.com/DataDog/datadog-agent/pull/8556/commits/2354a436ec591c7cbf376a72af4ef4ee95704c6d